tornado/auth.py

```diff
@@ -980,35 +1034,13 @@ def get(self):
         .. testoutput::
            :hide:
 
-        The given path is relative to ``self._FACEBOOK_BASE_URL``,
+        The given path is relative to ``self._OAUTH_BASE_URL``,
```

Comment:
Even though this was named as a semi-private attribute it was documented for application use so we shouldn't change it now. I think it's better to construct the url in `facebook_request` and `google_request` so we don't have to introduce another magic customizable class attribute.